jax/numpy/lax_numpy.py

```diff
@@ -2625,6 +2625,30 @@ def cov(m, y=None, rowvar=True, bias=False, ddof=None, fweights=None,
   return true_divide(dot(X, X_T.conj()), f).squeeze()
 
 
+@_wraps(onp.corrcoef)
+def corrcoef(x, y=None, rowvar=True, bias=None, ddof=None):
+  msg = ("jax.numpy.cov not implemented for nontrivial {}. "
```

Comment:
Two things:
a) this should probably say "corrcoef" not "cov".
b) can you please move the message into the `if` condition? No need to allocate a string in the non-error case. Alternatively, you could just omit the error completely, assuming that all we are missing is the corresponding code in `jax.numpy.cov`.